fh-android-sdk/pom.xml

```diff
@@ -26,6 +26,7 @@
   <properties>
     <loopj.version>1.4.8</loopj.version>
     <aerogear.android.push.version>2.2.1</aerogear.android.push.version>
+    <aerogear.android.store.version>3.0.0-SNAPSHOT</aerogear.android.store.version>
```

Comment:
> I don't feel we should be merging SNAPSHOT code into the master branch. A STABLE version should be required.

Of course not, that is why I added a `PS` about it. The idea is let people know what we are doing, review, ask, etc, and of course merge it in a branch until we don't release the dependency.

Sorry if I did not clear about it.